src/main/java/net/openhft/chronicle/wire/Wires.java

```diff
@@ -603,9 +603,6 @@ public static <E> E object0(ValueIn in, @Nullable E using, @Nullable Class clazz
     }
 
     public static <E> E object1(ValueIn in, @Nullable E using, @Nullable Class clazz, boolean bestEffort) throws InvalidMarshallableException {
-        if (using instanceof AbstractMarshallableCfg)
```

Comment:
undid this but not the associated test. I believe that the change to AbstractMarshallableCfg is more intentional